### DISCUSSION TITLE
2293 동전1, nodejs 문제

### DISCUSSION THREAD

**Discussion by steinjun0** [tony9402/baekjoon#445] [Question & Answer] [answered]
안녕하세요! 추천 문제 늘 잘 풀고 있습니다.

2293 동전1 문제가 node.js로 현재 풀리지 않는 것으로 보입니다(메모리 오류, 어떠한 코드를 넣어도 메모리 초과가 발생).  

이 문제는 여기서 커버할 범위를 넘어서는 것 같긴 합니다만, 제가 모르는 해결책이 있을까 해서 글 남깁니다. 혹시 다른 분들도 저처럼 nodejs로 시도 하실 수도 있으니까요!

**Comment by tony9402** [comment, marked as answer]
안녕하세요 ! 위 문제를 확인해본결과 node.js언어로 풀 수 없는 시간제한인 것 같습니다.

[1000번 문제 (A+B)](https://www.acmicpc.net/problem/1000)에서 node.js로 맞춘 기록을 봤는데 단순 입출력만 해도 상당한 메모리를 차지하는 것으로 보입니다.

이는 백준 온라인 저지(BOJ)를 관리해주시는 분이 특정 언어(ex. Node.js) 메모리 제한을 늘려주지 않는 이상 못푼다고 보시면 될 것 같습니다.

백준 게시판 공지를 보시면 ["Swift, node.js, Lua, Rust 추가 시간/메모리 제한"](https://www.acmicpc.net/board/view/71957) node.js 메모리 이슈가 있습니다. 해당 문제를 언급하시면 관지라님이 확인 후 추가메모리를 늘릴 것 같습니다

**Reply by steinjun0** [reply]
감사합니다!!
